### PR TITLE
MLPAB-1833: Show invited members

### PR DESCRIPTION
**Pull request by acsauk**
# Purpose

Fixes MLPAB-1833

### CODE REVIEW

File: cypress/e2e/supporter/invite-member.cy.js [hunk 1]
```diff
@@ -7,29 +7,35 @@ describe('Invite member', () => {
...
     cy.contains('button', 'Send invite').click();
 
-    cy.url().should('contain', '/invite-member-confirmation');
+    cy.url().should('contain', '/manage-organisation/organisation-details');
     cy.checkA11yApp();
-    cy.contains(TestEmail);
+
+    cy.get('.govuk-notification-banner--success').within(() => {
+      cy.contains((TestEmail))
+    });
```
Comment (hawx) [inline cypress/e2e/supporter/invite-member.cy.js:22]: or just `cy.contains('.govuk-notification-banner--success', TestEmail)`

File: web/template/supporter/organisation_details.gohtml [hunk 2]
```diff
@@ -20,16 +20,73 @@
 
       <h1 class="govuk-heading-xl">{{ tr .App "manageOrganisation" }}</h1>
 
-      <div class="govuk-tabs app-tabs-no-border">
+      <div class="govuk-tabs app-tabs-no-border" data-module="govuk-tabs">
```
Comment (hawx) [inline web/template/supporter/organisation_details.gohtml:23]: my plan for this (sorry, should have said/made it obvious) was to have them as real pages, that way when you change the organisation name you get redirected to the organisation details page/tab

File: web/template/supporter/organisation_details.gohtml [hunk 3]
```diff
@@ -20,16 +20,73 @@
...
+              </div>
+            </div>
+          {{ end}}
+
+          <a class="govuk-button" href="{{ link .App global.Paths.Supporter.InviteMember.Format }}">{{ tr .App "inviteTeamMember" }}</a>
+
+          {{ if .InvitedMembers }}
+            <h2 class="govuk-heading-m">{{ tr .App "invitedTeamMembers" }}</h2>
+
+            <dl class="govuk-summary-list app-summary-list--border-top">
```
Comment (hawx) [inline web/template/supporter/organisation_details.gohtml:55]: would a table be better? the last column is taking up a lot of space causing the email to wrap

File: web/template/supporter/organisation_details.gohtml (hunk 2 above)
Comment (hawx) [reply web/template/supporter/organisation_details.gohtml:23]: (also that is what I'm thinking for the donor/attorney/certificate provider dashboard when we get round to the pagination ticket)

Comment (acsauk) [reply web/template/supporter/organisation_details.gohtml:23]: Makes sense with the name change use case. I'll update 👍

File: web/template/supporter/organisation_details.gohtml (hunk 3 above)
Comment (acsauk) [reply web/template/supporter/organisation_details.gohtml:55]: I spent about 15 minutes trying to work out how to space this better for the email and gave up. I think I'd copied the markup from another list of things in the app but hadn't twigged it was for fewer props displayed. I'll switch to a table instead

File: internal/page/supporter/manage_team_members.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+type manageTeamMembers struct {
+	App            page.AppData
+	Query          url.Values
+	Errors         validation.List
+	Organisation   *actor.Organisation
+	InvitedMembers []*actor.MemberInvite
+}
+
+func ManageTeamMembers(tmpl template.Template, organisationStore OrganisationStore) Handler {
```
Comment (hawx) [inline internal/page/supporter/manage_team_members.go:21]: just need a basic set of tests for this now it is separate

**Review by hawx — APPROVED** [review]

**Pull request merged**